src/js/vue-plugins/router.js

```diff
@@ -176,4 +196,41 @@ const router = new Router({
   },
 });
 
+router.beforeEach((to, from, next) => {
```

Comment:
I wonder if this could somehow be handled through vue-meta which we already use?

Reply:
Definitly !!! Actually, we use `vue-head`. 

I looked for vue-meta, and as I did not found it, I did this small code (didn't want to add new dep).

Let me update this.